File: src/main/java/sawyern/cookiebot/exception/AssertMessage.java
package sawyern.cookiebot.exception;

import lombok.experimental.UtilityClass;

@UtilityClass
public class AssertMessage {
    public static final String EMPTY_MESSAGE = "Cannot parse empty message";
    public static final String DISCORD_CLIENT_NOT_SET = "Discord client not yet set!";

    public static final String DISCORD_TOKEN_NOT_SET = "Discord token was not provided in properties file";
    public static final String ERROR_LOADING_PROPERTY = "Error loading property";

}

File: src/main/kotlin/sawyern/cookiebot/exception/AssertMessage.kt
package sawyern.cookiebot.exception

class AssertMessage {
    companion object {
        val EMPTY_MESSAGE = "Cannot parse empty message"
        val DISCORD_CLIENT_NOT_SET = "Discord client not yet set!"

        val DISCORD_TOKEN_NOT_SET = "Discord token was not provided in properties file"
        val NULL_BOT_COMMANDS = "Error initializing bot command list"
        val ERROR_LOADING_PROPERTY = "Error loading property"
    }
}